saleor/core/notification/mutation_handler.py

```diff
@@ -0,0 +1,86 @@
+import json
+
+from ...account.models import User
+from ...core.permissions import AccountPermissions
+from ...core.permissions import OrderPermissions, ProductPermissions
+from ...order.models import Order
+from ...product.models import ProductVariant
+from ...webhook.payloads import (
+    generate_customer_payload,
+    generate_order_payload,
+    generate_product_variant_payload,
+)
+
+PAYLOAD_MAPPING = {
+    "ProductVariant": (
```

Comment:
The assumption of this mapping is to attach a defined object to the payload, which will be sent via webhook notify/ or by plugins. 
We want this mapping for User and for Order because these objects have enough details about customer which we can use to send notification (name, email, phone etc). ProductVariant doesn't have such details so there is no need to add it here.